backend/code_review_backend/issues/management/commands/cleanup_issues.py

```diff
@@ -29,24 +29,39 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         clean_until = timezone.now() - timedelta(days=options["nb_days"])
+
+        # For issue and issuelink we can use _raw_delete since it's faster and doesn't
+        # give OOM since there are no CASCADE directives in the model since django
+        # doesn't need to query the model and compute the dependencies.
+        issues_to_delete = Issue.objects.filter(created__lte=clean_until)
+        count_issues = issues_to_delete.count()
+
+        if count_issues:
+            issuelinks_to_delete = IssueLink.objects.filter(
+                issue_id__in=issues_to_delete
+            )
+            logger.info(
+                f"Retrieved {issuelinks_to_delete.count()} old issue_links to be deleted."
+            )
+            issuelinks_to_delete._raw_delete(issuelinks_to_delete.db)
+            logger.info(f"Retrieved {count_issues} old issues to be deleted.")
+            issues_to_delete._raw_delete(issues_to_delete.db)
+        else:
+            logger.info("Didn't find any old issues to delete.")
+
         rev_to_delete = Revision.objects.filter(
             base_repository__slug__in=["autoland", "mozilla-central"],
             head_repository__slug__in=["autoland", "mozilla-central"],
             created__lte=clean_until,
         )
 
-        count = rev_to_delete.count()
-        if not count:
+        count_rev = rev_to_delete.count()
+        if count_rev:
```

Comment:
Let's keep the old behavior of early return to minimize the diff and to keep the rest of the code with a smaller indentation